normalizer/tests/test_api_comparison.py

```
@@ -0,0 +1,171 @@
+        if gsi_lat and jageocoder_lat:
+            distance = haversine(gsi_lat, gsi_lon, jageocoder_lat, jageocoder_lon)
+            print(f"  国土地理院: {gsi_lat:.6f}, {gsi_lon:.6f}")
+            print(f"  Jageocoder: {jageocoder_lat:.6f}, {jageocoder_lon:.6f}")
+            print(f"  座標間の距離: {distance:.1f}m")
+            
+            if distance < 1.0:
+                print("  [WARNING] 座標が完全に一致（同じデータソースの可能性）")
+            elif distance < 10.0:
+                print("  [WARNING] 座標がほぼ一致（同じデータソースの可能性が高い）")
+            else:
+                print(f"  [OK] 座標に差異あり（別々のデータソース）")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)